examples/Vanilla/OnLoad/onload.js

```diff
@@ -4,11 +4,20 @@ function rciMainAction(tenancyId, rciSdk) {
   const transport = new rciSdk.Transport(targetUrl);
 
   // Step 2: Capture your default collectors
-  const defaults = rciSdk.collector.defaultCollectors;
+  let defaults = rciSdk.collector.defaultCollectors;
+  const config = {
```

Comment:
@robertdumitrescu CONSISTENCY - I'm all for composition, leading to maximum flexibility.  However, if we're going through the trouble of configuring the collectors, I'd like to see all the concatenation and registering to be handled. For example, if something like the following is provided:

```
const config = {
   transport: new rciSdk.Transport(targetUrl),
   onload: {
     defaults: true,
     enabled: true,
     custom: [...]
   },
   actions: {
     defaults: true,
     enabled: true,
     custom: [...]
   }
}
rciSdk.ConfigurationService.run(config);
``` 

That should register all triggers and collectors with the relevant producer.  Job done!